packages/jumbotron-carousel/carouselTimer.js

```diff
@@ -1,36 +1,47 @@
-document.addEventListener('DOMContentLoaded', () => {
+if (window.NodeList && !NodeList.prototype.forEach) {
+    NodeList.prototype.forEach = Array.prototype.forEach;
+}
+
+document.addEventListener('DOMContentLoaded', function () {
     const checkboxes = document.querySelectorAll('.gb-carousel__activator');
     const autoplayDelay = document.getElementById('autoplay-value').value * 1000;
     if (autoplayDelay && autoplayDelay > 0) {
         callTimer();
     }
 
-    document.querySelectorAll('.gb-carousel__manual-next').forEach(element => {
-        element.addEventListener('click', (event) => {
+    document.querySelectorAll('.gb-carousel__manual-next').forEach(function (element) {
+        element.addEventListener('click', function (event) {
             event.preventDefault();
             checkboxes[getNextArrayId()].checked = true;
         });
     });
 
-    document.querySelectorAll('.gb-carousel__manual-prev').forEach(element => {
-        element.addEventListener('click', (event) => {
+    document.querySelectorAll('.gb-carousel__manual-prev').forEach(function (element) {
+        element.addEventListener('click', function (event) {
             event.preventDefault();
             checkboxes[getPrevArrayId()].checked = true;
         });
     });
 
     function callTimer() {
-        setTimeout(_=>{
+        setTimeout(function () {
             checkboxes[getNextArrayId()].checked = true;
             callTimer();
         }, autoplayDelay);
     }
 
     function getCurrentArrayId() {
         const currentItem = document.querySelector('.gb-carousel__activator:checked');
-        return Array.from(checkboxes).indexOf(currentItem);
+
+        for (let i = 0; i <= checkboxes.length; i++) {
```

Comment:
I would use `checkboxes.indexOf` instead of doing a loop manually.
Or at least, return -1 if the index is not found to have the same behaviour.

Reply:
.indexOf is not supported in IE11. 

I will return -1, no problem.

Reply:
It is supported since IE9
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf